applications/luci-app-upnp/luasrc/view/upnp_status.htm

```diff
@@ -41,6 +42,7 @@ <h3><%:Active UPnP Redirects%></h3>
 			<div class="th"><%:Protocol%></div>
 			<div class="th"><%:External Port%></div>
 			<div class="th"><%:Client Address%></div>
+			<div class="th"><%:HostName%></div>
```

Comment:
Maybe just call this one `Host` as there can be either a hostname, or a MAC or nothing... depending on available info.